src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
+    blue,
+    green,

```

Comment:
```suggestion
    RED,
    BLUE,
    GREEN,
    ...
```
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names